project/AppDependencies.scala

```diff
@@ -15,6 +15,7 @@ object AppDependencies {
 
   val test: Seq[ModuleID] = Seq(
     "org.scalatest"               %% "scalatest"                % "3.2.12",
+    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-28"  %"0.73.0",
```

Comment:
I dont see you using anything from this test library if you are not using it please delete it or if you want to add it here please add tests for repository which is better